apps/mason2/simulate_sanger.cpp

```diff
@@ -58,6 +58,9 @@ void _simulateSequence(TRead & read, TRng & rng, TFrag const & frag,
     typedef typename seqan2::Iterator<TFrag>::Type TFragIter;
     TFragIter it = begin(frag, seqan2::Standard());
 
+    std::uniform_int_distribution<unsigned short> dist{};
+    using param_t = typename std::uniform_int_distribution<unsigned short>::param_type;
+
```

Comment:
Since this is all copy'n'paste code (😢), the same comments from above apply to this part.

Reply:
I'll check whether I can put that function in some header